Ifzan/binary.js

```diff
@@ -0,0 +1,31 @@
+function binary (arr,x,y,find)
+{
+    if(Array.isArray(arr) !== true)
+    {
+    console.log('Invalid');
+    }
+    else if(x>y)
```

Comment:
not sure why we need this

Reply:
to check for empty array. if the front is greater than rear then the array is empty